D/DASKR/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "DASKR"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://www.netlib.org/ode/daskr.tgz", "5472e98c954062ea69d19bd16d1cb8a8835497d6b0df18fe5753bae61d151774")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd DASKR
+install_license LICENSE
+[ -d $libdir ] || mkdir $libdir
+gfortran -shared -fPIC -o $libdir/daskr.${dlext} solver/d*.f
```

Comment:
Maybe call it libdaskr?

Reply:
The [archive source](https://www.netlib.org/ode) is named just `daskr.tgz` and DASKR.jl uses it as daskr.dll. If you insist I'm fine with changing it to `libdaskr` however